db/qmarks_test.go

```diff
@@ -0,0 +1,11 @@
+package db
+
+import "testing"
+import "github.com/letsencrypt/boulder/test"
+
+func TestQuestionMarks(t *testing.T) {
+	test.AssertEquals(t, QuestionMarks(0), "")
```

Comment:
Would we want to consider having `QuestionMarks(0)` result in an error of some kind? It seems like the kind of corner case where all these tests would pass but the end result would be invalid SQL syntax (e.g. "INSERT INTO certificateNames (a, b, c) VALUES () WHERE ..."). Probably not worth the friction of having this return an error but just wanted to think about it.

Reply:
I think this would be a reasonable place for a panic!

Reply:
Oh yeah that makes sense! "0 through -Inf are invalid inputs".